### PR TITLE
Want a list of years, not just most recent.

### PR DESCRIPTION
**Pull request by LindsayYoung**
Modeling this off the election cycles behavior, which is working.

### CODE REVIEW

File: data/sql_updates/create_candidates_view.sql [hunk 1]
```diff
@@ -14,7 +14,7 @@ select
         else 'Unknown' end as candidate_status_full,
     max(dimoffice.office_district) as district,
     max(co.cand_election_yr) as active_through,
-    array_agg(distinct co.cand_election_yr)::int[] as election_years,
+    array_agg(distinct dcp.cand_election_yr)::int[] as election_years,
```
Comment (jmcarp) [inline data/sql_updates/create_candidates_view.sql]: That column doesn't exist on `dimcandproperties`.

Comment (LindsayYoung) [inline data/sql_updates/create_candidates_view.sql]: Yea, it is just election_year. sorry

**Comment by jmcarp** [comment]
A few thoughts about this:

* In some sense, this was fixed in #770, which restored multiple values for `election_years` for both candidate views
* If `dimcandproperties.election_yr` is a better choice that `dimcandoffice.cand_election_year`, then I'm in favor of this patch, in which case we should also do the following:
    * Also use `dimcandproperties.election_yr` to calculate `active_through`
    * Remove the now-unnecessary join on `dimcandoffice`
    * Apply all of the above changes to the candidate detail view as well

**Comment by LindsayYoung** [comment]
I just refreshed on dev and got the right results. was the problem that the fix in #770 just hadn't deployed yet?

**Pull request closed without merging**